websites/docs-smoke-test/src/release-notes/relase-format-definition.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Release Note Format Example
+date: 2020-02-28
+type: New Feature
+topics:
+  - Products
+  - Product List
+#  - Categories
+#  - Uncomment-unused
+#  - Products
+#  - Product List
+#  - Products
+#  - Product List
+description: |
+  Write Release Notes in the new structured format and see them immediately published in RSS feeds,
+  the Merchant Center and on twitter.com.
+---
+
+The File name can be chosen arbitrarily and subfolders. Only the content of the file has a meaning.
```

Comment:
```suggestion
Choose any file name you want. Only the content of the file has a meaning.
```

... not sure what you meant with the subfolders bit. That it can be in any folder?

Reply:
Yes, I suggest to allow any folder structure inside `release-notes` so you can e.g. later archive stuff into `release-notes/2019-Q1` or whatever structure the website maintainer likes without affecting how the site renders.   

The current file list is pretty long and starts to get annoying.